shells/pipes-shell/web/deploy/deploy.sh

```diff
@@ -11,3 +11,8 @@ ln -s $PWD/../../../../particles $PWD/dist/particles
 cp -fR ../../../lib/build/worker.js dist/
 # collate sources
 npx webpack --display=errors-only
+# Arcs cache manager and versioning
+# Must be done at the last step to ensure the correct dist/** checksum
+cp -fR ../cache-mgr*.js dist/
+sed -i "s/__ARCS_MD5__/$(tar -cf - dist | md5sum | cut -d' ' -f1)/g" \
+dist/cache-mgr-sw.js
```

Comment:
Is cache-mgr used only for Android? If so, there's no need to update this script, we only use android_deploy.sh for android.

Reply:
I used the remote pipes-shell too thus changed the script as well.